tests/whitebox/_misc.py

```diff
@@ -112,6 +112,10 @@ def check_error(self, expected_cause, command_line, expected_code):
         cause = exception.__cause__
         self.assertIsInstance(cause, expected_cause)
 
+        error_string = str(exception)
+        self.assertNotEqual(error_string, None)
```

Comment:
```self.assertNotEqual(error_string, "")``` here. ```None``` has type ```NoneType```, so if ```error_string``` is an instance of a ```str``` it will certainly not equal ```None```.